config/tracks/acu_bathurst.ini

```diff
@@ -128,48 +128,37 @@ VALUE_0 = 1, 1, 1, 4
 VALUE_0_OFF = 0, 0, 0
 ACTIVE = 1
 
-[MATERIAL_ADJUSTMENT_10]
-
+[SHADER_REPLACEMENT_...]
 MATERIALS=GRASS38?
 KEY_0=ksDiffuse
 VALUE_0=0.2
-OFF_VALUE_0=0.25
 KEY_1=ksAmbient
 VALUE_1=0.40
-OFF_VALUE_1=0.25
-
-[MATERIAL_ADJUSTMENT_11]
 
+[SHADER_REPLACEMENT_...]
 MATERIALS=3Dgrass?
 KEY_0=ksDiffuse
 VALUE_0=0.2
-OFF_VALUE_0=0.25
 KEY_1=ksAmbient
 VALUE_1=0.10
-OFF_VALUE_1=0.25
-
-[MATERIAL_ADJUSTMENT_12]
 
+[SHADER_REPLACEMENT_...]
 MATERIALS=gumtree02,gumtree03,treeline,WINEPLANTS_LO1,trees_a,bush01?
 KEY_0=ksDiffuse
 VALUE_0=0.175
-OFF_VALUE_0=0.25
 KEY_1=ksAmbient
 VALUE_1=0.25
-OFF_VALUE_1=0.25
 
-[MATERIAL_ADJUSTMENT_13]
 
+[SHADER_REPLACEMENT_...]
 MATERIALS=terrain1?
 KEY_0=ksDiffuse
 VALUE_0=0.2
-OFF_VALUE_0=0.25
 KEY_1=ksAmbient
 VALUE_1=0.22
-OFF_VALUE_1=0.25
 
-[MATERIAL_ADJUSTMENT_14]
 
+[SHADER_REPLACEMENT_...]
 MATERIALS=gumtree01?
 KEY_0=ksDiffuse
 VALUE_0=0.20
```

Comment:
I think there was a bit of a misunderstanding here, entirely my fault.

Read the documentation for shader replacements [here](https://github.com/ac-custom-shaders-patch/acc-extension-config/wiki/General-%E2%80%93-Shader-replacements), they don't share the same syntax as material adjustments.

Reply:
really? I've tested these changes during runtime and they applied fine, can change if you prefer, however.